libs/coin-polkadot/src/account.ts

```diff
@@ -67,36 +67,32 @@ function formatAccountSpecifics(account: PolkadotAccount): string {
     formatCurrencyUnit(unit, account.spendableBalance, formatConfig) +
     " spendable. ";
 
-  if ((polkadotResources as PolkadotResources).lockedBalance.gt(0)) {
+  if (polkadotResources.lockedBalance.gt(0)) {
     str +=
-      formatCurrencyUnit(
-        unit,
-        (polkadotResources as PolkadotResources).lockedBalance,
-        formatConfig
-      ) + " locked. ";
+      formatCurrencyUnit(unit, polkadotResources.lockedBalance, formatConfig) +
+      " locked. ";
   }
 
-  if ((polkadotResources as PolkadotResources).unlockedBalance.gt(0)) {
+  if (polkadotResources.unlockedBalance.gt(0)) {
     str +=
       formatCurrencyUnit(
         unit,
-        (polkadotResources as PolkadotResources).unlockedBalance,
+        polkadotResources.unlockedBalance,
         formatConfig
       ) + " unlocked. ";
   }
 
-  if ((polkadotResources as PolkadotResources).stash) {
-    str += "\nstash : " + (polkadotResources as PolkadotResources).stash;
+  if (polkadotResources.stash) {
+    str += "\nstash : " + polkadotResources.stash;
   }
 
-  if ((polkadotResources as PolkadotResources).controller) {
-    str +=
-      "\ncontroller : " + (polkadotResources as PolkadotResources).controller;
+  if (polkadotResources.controller) {
+    str += "\ncontroller : " + polkadotResources.controller;
   }
 
-  if ((polkadotResources as PolkadotResources).nominations?.length) {
+  if (polkadotResources.nominations?.length) {
     str += "\nNominations\n";
-    str += ((polkadotResources as PolkadotResources).nominations as any[])
+    str += (polkadotResources.nominations as any[])
```

Comment:
what's the behavior when you add an array to a string ?

Reply:
I have no idea what's happening in this case

But i don't want to change anything functional and it's just about types and not repeating casts